src/components/DoctorCard/DoctorCard.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import { Button, Collapse } from "@material-ui/core";
+import DoctorCardStyle from "./DoctorCard.style";
+import { IDoctorCard } from "./DoctorCard.interface";
+import { useRequest } from "hooks/useRequest";
+import { AddressDetails } from "components/AddressDetails";
+import { DoctorAvatar } from "components/DoctorAvatar";
+import { DoctorAvailability } from "components/DoctorAvailability";
+
+const DoctorCard: React.FC<IDoctorCard> = (props) => {
+  const { id, address, name } = props;
+  const classes = DoctorCardStyle();
+  const [open, setOpen] = useState(false);
+  const { apiData, isLoading, execute } = useRequest("GET", `availabilities?doctorId=${id}`, false);
```

Comment:
Good idea. Without context, `apiData` is not self explanatory, we do not know what data it is. Maybe availabilities is a better name.
```suggestion
  const { apiData: availabilities, isLoading, execute } = useRequest("GET", `availabilities?doctorId=${id}`, false);
```

Reply:
I totally agree! thanks for that